### ISSUE TITLE
Accessibility: ColorCard list contains <p> tags

### ISSUE THREAD

**Issue by nhsbsa-wicoo** [nhsbsa/nhsbsa-digital-playbook#495]
## Summary

Some lists contain child elements that are not allowed. 

When content elements other than list items are contained within a set of list elements, screen readers cannot inform the listener that they are listening to items within the list.

You can replicate by scanning the page with the Axe chrome plugin.

## Affected pages
- [Nunjucks](https://nhsbsa.github.io/nhsbsa-digital-playbook/how-to-contribute/nunjucks/)

## User impact

This issue primarily affects:

- Screen reader users
- Users may not be aware that related items form a list and may miss contextual information such as the number of items available.

## WCAG 2.2 Success Criteria
1.3.1 Info and Relationships
